### DISCUSSION TITLE
Change nvim-tree background color

### DISCUSSION THREAD

**Discussion by santigo-dev** [rose-pine/neovim#53] [Q&A] [answered]
I'm using this [tree plugin](https://github.com/kyazdani42/nvim-tree.lua) and would like to have the background of the tree window on a slightly light color (I'm using the default colorscheme, the darkest one), but I don't know how to, other colorschemes offer this by default, how would I change it to something like #26233A?

**Comment by mvllow** [comment, marked as answer]
i'd suggest this pattern for using the rosé pine palette

```lua
local palette = require('rose-pine.palette')

vim.cmd(string.format('autocmd ColorScheme * hi NvimTreeNormal guibg=%s', palette.foam))
```

this may be added as an option in the future 😌
